galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -140,10 +140,11 @@ class CollectionUploadViewSet(LocalSettingsMixin, pulp_ansible_views.CollectionU
     ]
     parser_classes = [AnsibleGalaxy29MultiPartParser]
 
-    def _dispatch_import_collection_task(self, artifact_pk, repository=None, **kwargs):
+    def _dispatch_import_collection_task(self, temp_file_pk, repository=None, **kwargs):
         """Dispatch a pulp task started on upload of collection version."""
-        locks = [str(artifact_pk)]
-        kwargs["artifact_pk"] = artifact_pk
+        locks = [str(temp_file_pk)]
```

Comment:
I believe temp_file does not need a lock, @bmbouter wdyt?

Reply:
pulp_ansible's impl of _dispatch_import_collection_task() doesn't seem to use.

That may be a left over from before I figured out the share /var/lib/pulp/ issues.
Seeing if I can remove it.

Reply:
Doesn't seem to need a lock. Updated.